README.md

```diff
@@ -1,13 +1,84 @@
 # Project Happy Thoughts API
```

Comment:
Great job on the creation of your readme! It looks great, explains all parts of the API for the user whilst also meeting explaining the project at hand. Love it!